openapi.yaml

```
@@ -663,6 +686,24 @@ components:
           description: The unique identifier of the signer
           type: string
 
+    SignerTickerMessage:
+      description: represents an
```

The field description is not completely filled.
```suggestion
      description: represents a SignerTickerMessage
```
I just suggested 'SignerTickerMessage' as it will probably be modified later.